src/validators/default-value-validator.ts

```diff
@@ -5,8 +5,11 @@ import { codedTypes } from '../constants';
 export const DefaultFieldValueValidator: FormFieldValidator = {
   validate: (field: FormField, value: any) => {
     if (codedTypes.includes(field.questionOptions.rendering)) {
+      const valuesToCheck = Array.isArray(value) ? value : [value];
       // check whether value exists in answers
-      if (!field.questionOptions.answers?.find((answer) => answer.concept == value)) {
+      if (
+        !valuesToCheck.some((val: string) => field.questionOptions.answers?.find((answer) => answer.concept === val))
```

Comment:
`some(...)` tests whether at least one element passes the condition specified in the callback, you want to use `every(...)`

```ts
!valuesArray.every((value: string) => field.questionOptions.answers.includes(value));
```

Reply:
@samuelmale switched to` every(...)` and maintained the callback condition to check if for every val in `valuesToCheck`, there exists at least one answer in `field.questionOptions.answers` where `answer.concept` is equal to `val`

Reply:
@gitcliff any reason as to why you're using `find` as opposed to using `includes`?

```ts
!valuesArray.every((value: string) => field.questionOptions.answers.includes(value));
```

Reply:
@samuelmale when i use includes the validation fails , secondly the elements in the `valuesArray` are concepts UUIDs. 
field.questionOptions.answers array holds ie```
 {
                        "concept": "30e2da8f-34ca-4c93-94c8-d429f22d381c",
                        "label": "Congestion"
                      }
```